src/pages/SwapV2/index.tsx

```diff
@@ -656,7 +656,7 @@ export default function Swap({ history }: RouteComponentProps) {
   }, [qs, history])
 
   // swap?inputCurrency=xxx&outputCurrency=yyy. xxx yyy not exist in chain => remove params => select default pair
-  const checkParamWrongDebounce = useCallback(debounce(checkParamWrong, 300), [])
+  const checkParamWrongDebounce = useMemo(() => debounce(checkParamWrong, 300), [checkParamWrong])
```

Comment:
Is there any reason that this is changed from useCallback to useMemo?

Reply:
useCallback is designed for inline function only 

useCallback(fn, deps) is equivalent to useMemo(() => fn, deps).

Reply:
Make sense